hoot-services/pom.xml

```diff
@@ -136,7 +136,7 @@
                     parallelism, but haven't been able to get them to work...so forking instead. -->
                     <!-- parallel>methods</parallel>
                     <useUnlimitedThreads>true</useUnlimitedThreads -->
-                    <forkCount>4</forkCount>
+                    <forkCount>16</forkCount>
```

Comment:
@brianhatchl  @bwitham 

It was intentional.  I hardcoded '4' thinking that it should suffice as a general case.  However, we can always make this more flexible and read in $(nproc) as an external property.